lib/compat.php

```diff
@@ -102,6 +102,37 @@ function gutenberg_shim_fix_api_request_plain_permalinks( $scripts ) {
 }
 add_action( 'wp_default_scripts', 'gutenberg_shim_fix_api_request_plain_permalinks' );
 
+/**
+ * Shims support for emulating HTTP/1.0 requests in wp.apiRequest
+ *
+ * @see https://core.trac.wordpress.org/ticket/43605
+ *
+ * @param WP_Scripts $scripts WP_Scripts instance (passed by reference).
+ */
+function gutenberg_shim_api_request_emulate_http( $scripts ) {
+	$api_request_fix = <<<JS
+( function( wp ) {
+	var oldApiRequest = wp.apiRequest;
+	wp.apiRequest = function ( options ) {
+		if ( options.method ) {
+			if ( [ 'PATCH', 'PUT', 'DELETE' ].indexOf( options.method.toUpperCase() ) >= 0 ) {
+				if ( ! options.headers ) {
+					options.headers = [];
```

Comment:
This should be assigned as an object, not array.

http://api.jquery.com/jquery.ajax/ (`settings.headers`)

Reply:
Missed that, but array is also an object in JS... so might work

Reply:
👍🏻